flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
that's disappointing.

Reply:
I'm not totally close yet to have an ARM port but need to get a device first to test it out